frontend/src/components/SubmitProblem.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { ChangeEvent } from "react";
+import "./styles/submitProblem.css";
+/**
+ * SubmitProblem component is a Popup Menu that contains input field and submit button. The component creates new Thread.
+ * @param {{changeText: function, show: boolean, inputText: string, submitThread: function}} props
+ * @param {function} changeText function that changes state of the input field text
+ * @param {string} inputText text of the input field
+ * @param {function} submitThread function that submits new thread
+ */
+
+type Props = {
+  changeText: (value: string) => void;
+  inputText: string;
+  submitThread: () => void;
+  toggleNotification?: (value: boolean) => void;
+};
+
+export default function SubmitProblem({
+  changeText,
+  inputText,
+  submitThread,
+  toggleNotification,
+}: Props) {
+  function getText(event: ChangeEvent<HTMLInputElement>) {
+    changeText(event.target.value);
+  }
+  return (
+    <React.Fragment>
+      <span className="notification-text">Describe your problem</span>
+      <div className="message-wrapper notification">
+        <input
+          placeholder="Type your message here..."
+          onChange={getText}
+          value={inputText}
+        />
+        <div className="button-attach">
+          <svg
```

Comment:
If you don't apply any styles for this svg, and don not interact with this using js, then you can move svg in separate file, and paste this here using img tag:
<img src={require("path")} alt="name"/>

Reply:
Unfortunately, I apply some styling to svgs, therefore it would be better to leave them as it is.